ocs_ci/ocs/ui/workload_ui.py

```diff
@@ -0,0 +1,492 @@
+import copy
+import logging
+import random
```

Comment:
Why workload_ui file located in ocs_ci/ocs/ui folder? we dont use any UI automation here

Reply:
I put it as a helper for several UI tests I automate, this workloads are only for UI tests, I really don't want to universalize and maintain the workload_ui for non-ui tests.